Auth/class.LDAPAuthenticator.php

```diff
@@ -190,11 +203,13 @@ public function login($username, $password)
         }
         $user = $user[0];
         $server->unbind();
+
         $ret = $server->bind($user->dn, $password);
         if($ret !== false)
         {
-            return array('res'=>true, 'extended'=>$user); 
+            return array('res'=>true, 'extended'=>$user);
         }
+        $server->unbind();
```

Comment:
is there a time where we want to bind as the user except to validate their password?
is there a reason not to unbind the user?
explicitly unbinding would prevent another call being performed as the bound user inadvertently.. creating a potentially difficult to troubleshoot issue..